Google.PowerShell/Logging/GcLogSinkCmdlets.cs

```diff
@@ -221,16 +225,6 @@ protected override void ProcessRecord()
                 after: null,
                 otherFilter: Filter);
 
-            if (Before.HasValue)
-            {
-                logSink.EndTime = XmlConvert.ToString(Before.Value, XmlDateTimeSerializationMode.Local);
```

Comment:
I feel like we should have at least one release where we deprecate these parameters by writing a warning about the parameters.